src/geom/Transform.cc

```diff
@@ -206,18 +209,84 @@ std::ostream &operator<<(std::ostream &os, Transform<FromEndpoint, ToEndpoint> c
     return os;
 };
 
+namespace {
+
+class TransformPersistenceHelper {
+public:
+    table::Schema schema;
+    table::Key<table::Array<std::uint8_t>> bytes;
+
+    static TransformPersistenceHelper const & get() {
+        static TransformPersistenceHelper instance;
+        return instance;
+    }
+
+    // No copying
+    TransformPersistenceHelper(TransformPersistenceHelper const &) = delete;
+    TransformPersistenceHelper& operator=(TransformPersistenceHelper const &) = delete;
+
+    // No moving
+    TransformPersistenceHelper(TransformPersistenceHelper&&) = delete;
+    TransformPersistenceHelper& operator=(TransformPersistenceHelper&&) = delete;
+
+private:
+    TransformPersistenceHelper() :
+        schema(),
+        bytes(schema.addField<table::Array<std::uint8_t>>("bytes", "an opaque bytestring containing hte transform", ""))
```

Comment:
"hte" -> "the". Please say more about the format. It is that returned by Transform.writeString, so perhaps "a bytestring containing the output of Transform.writeString"? SkyWcs says "wcs string representation" which is a bit lean, but at least hints at the format.

Reply:
Is it desirable for the user to think they can interpret the persistence format without our code?  That's essentially exposing a new interface, which makes it harder to change that interface in the future.

I think that's unlikely to actually cause a problem in this case, but I do think it's worth thinking about whether opacity is a virtue here.

Reply:
`Transform.writeString` is explicitly is intended for persistence so I think it is safe in this case. I lean towards documenting our use of that format, but I won't insist.